src/lib/layer-manager.js

```diff
@@ -428,3 +537,33 @@ function layerName(layer) {
   }
   return !layer ? 'null layer' : 'invalid layer';
 }
+
+/**
+ * Get mouse position {x, y} relative to the container
+ */
+function getPos(event) {
```

Comment:
Abbreviated names are typically avoided in this code base: `getPos` => `getPosition`